docs/basic-features/image-optimization.md

```diff
@@ -100,7 +100,15 @@ The following describes the caching algorithm for the default [loader](#loader).
 
 Images are optimized dynamically upon request and stored in the `<distDir>/cache/images` directory. The optimized image file will be served for subsequent requests until the expiration is reached. When a request is made that matches a cached but expired file, the cached file is deleted before generating a new optimized image and caching the new file.
 
-The expiration (or rather Max Age) is defined by the upstream server's `Cache-Control` header.
+The expiration (or rather Max Age) is defined by the upstream server's `Cache-Control` header or by adding configuration to `next.config.js`
+
+```js
+module.exports = {
+  images: {
+    maxAge: 60,
```

Comment:
Maybe we should call this `mimimumMaxAge`. It sounds weird but its clear that the value is only used as the lower bound. If the upstream image has a larger max-age then it will use that value instead of the config value.

Reply:
Another alternative is `minimumCacheSeconds`